lib/recurly.js

```diff
@@ -35,6 +36,10 @@ const defaults = {
   publicKey: '',
   parent: true,
   cors: false,
+  fraud: {
+    dataCollector: false,
```

Comment:
It would be nice to expose less of the internals of fraud detection in the configuration api. Is there a name for this that would be more meaningful for a novice implementer? Perhaps something that directly matches any fraud configuration the merchant does in the app.